src/Akeneo/UserManagement/Bundle/Controller/Rest/UserController.php

```diff
@@ -301,30 +307,36 @@ private function getUserOr404($identifier): UserInterface
     private function updateUser(UserInterface $user, array $data): JsonResponse
     {
         $previousUserName = $data['username'];
-        $passwordViolations = $this->validatePassword($user, $data);
-        if ($this->isPasswordUpdating($data) && $passwordViolations->count() === 0) {
-            $data['password'] = $data['new_password'];
+        if ($this->isPasswordUpdating($data)) {
+            $groups = ['UpdatePassword'];
```

Comment:
I think that you need to add the `Default` group to also validate the other properties of the User entity.
```suggestion
            $groups = ['Default', 'UpdatePassword'];
```